### ISSUE TITLE
support translated/_i18n fields in model.Meta.ordering

### ISSUE THREAD

**Issue by jieter** [zostera/django-modeltrans#25]
If `ordering` is defined in `Model.Meta`, the values doesn't pass through the standard queryset interface and probably doesn't allow using the `<field>_i18n` version of the field, which does make a lot of sense while ordering translated data.

Example:
```
from django.db import models
from modeltrans.fields import TranslationField

class Blog(models.Model):
    name = models.CharField(max_length=100)
    body = models.TextField()
    
    i18n = TranslationField(fields=('name', 'body'))

    class Meta:
        ordering = ('name', )
```

**Comment by jieter** [comment]
Django 2.0 adds expressions in ordering. That makes fixing this doable.

Pull request adding expression support for model.Meta.ordering: django/django#8673

**Comment by jieter** [comment]
fixed (for django 2.0) in 59b513d4b5becebf6837f7f8974ecabdd8c05557
